lib/Bio/KBase/GenomeAnnotation/GenomeAnnotationImpl.pm

```diff
@@ -26114,18 +26114,19 @@ sub run_pipeline
 
 	    if ($@)
 	    {
-		if ($stage->{failure_is_not_fatal})
-		{
-		    warn "Error invoking method $method: $@\nContinuing because failure_is_not_fatal flag is set";
+#		if ($stage->{failure_is_not_fatal})
+#		{
+#		    warn "Error invoking method $method: $@\nContinuing because failure_is_not_fatal flag is set";
```

Comment:
It's better to remove code completely than comment it out -- git will contain the previous versions of the code if people need to look back at it, and having a load of commented-out lines increases cognitive load on those looking through the codebase. Ideally code should be easy to read and understandable to all.